src/App.js

```diff
@@ -1,12 +1,16 @@
 import logo from "./logo.svg";
 import "./App.css";
 
-const PassingProsComponent = ({ children, ...props }) => (
-  <div>
-    {children}
-    {...props}
-  </div>
-);
+const PassingProsComponent = (props) => {
+  const { isDisabled, ...rest } = props;
+  return (
+    <div {...rest}>
+      {isDisabled ? 'Disabled' : 'Enabled'}
+    </div>
+  );
+};
+  );
+};
```

Comment:
**Suggestion:** Remove the duplicate closing brackets on lines 12 and 13. They are unnecessary and will cause a syntax error.
```suggestion

```